css/example/Image.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image</title>
+    <link rel="stylesheet" href="../components/Image.css">
+</head>
+<body>
+
+    <section class="image-round">
+            <img class="size-xl round" src="https://images.unsplash.com/photo-1643188441507-9b8ccc72f755?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDI3fDZzTVZqVExTa2VRfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+            <img class="size-xl round" src="https://images.unsplash.com/photo-1643620772691-e9d89e73a292?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMxfDZzTVZqVExTa2VRfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
```

Comment:
Try not to use the direct url of image. Instead save the image in a separate folder/file and import it in your file.

Reply:
It's just a demo component, I will make it easy for reviewers to review the code.
I will make folders/ file and import stuff in future